src/devices/Bmx280/BmxBase.cs

```diff
@@ -342,7 +333,7 @@ private long CompensatePressure(int adcPressure)
         /// <returns>
         ///  Value from register
         /// </returns>
-        private byte Read8BitsFromRegister(byte register)
+        internal byte Read8BitsFromRegister(byte register)
```

Comment:
Same comment about making internal. I don't see why you would change these so they should all remain private.

Reply:
yes, will switch back to private.